packages/legacy-client/src/ontology-runtime/aggregations/Aggregations.ts

```diff
@@ -423,3 +423,21 @@ export function visitInternalBucketing<
       throw new Error(`Unhandled bucketing: ${JSON.stringify(bucketing)}`);
   }
 }
+
+export interface QueryAggregation {
+  key: any;
+  value: any;
+}
+
+export interface NestedQueryAggregation {
+  key: any;
+  groups: Array<QueryAggregation>;
+}
+
+export interface QueryTwoDimensionalAggregation {
+  groups: Array<QueryAggregation>;
+}
+
+export interface QueryThreeDimensionalAggregation {
+  groups: Array<NestedQueryAggregation>;
+}
```

Comment:
Are we good with this copypaste here?

Reply:
This should be fine, it's just for mapping the aggregation responses from queries which look and behave a bit different than the ones the gateway returns :(